minicluster/pom.xml

```diff
@@ -43,10 +43,6 @@
       <groupId>commons-io</groupId>
       <artifactId>commons-io</artifactId>
     </dependency>
-    <dependency>
-      <groupId>commons-lang</groupId>
-      <artifactId>commons-lang</artifactId>
```

Comment:
Removing this is a great thing... in other modules, we can probably update to commons-lang3. We probably don't need commons-lang (pre-3) at all, at least not as a direct dependency.

Reply:
Checkout e0a8c17